src/Components/GameBoard.js

```diff
@@ -32,7 +34,7 @@ const reactLocal = "http://localhost:3000/"
 const remote = "https://data-structures-game.herokuapp.com/";
 
 //can also be const url = local; or const url = reactLocal;
-const url = remote;
+const url = local;
```

Comment:
Please change the URL to remote, or please help us find permanent solution for this.